examples/animation/pause_resume.py

```diff
@@ -3,6 +3,14 @@
 Pausing and Resuming an Animation
 =================================
 
+.. note::
```

Comment:
I would put the note below the description.

Reply:
On all of them?

Where would you put it on the [interactive functions example](https://79-21548241-gh.circle-artifacts.com/0/doc/build/html/gallery/event_handling/ginput_manual_clabel_sgskip.html#sphx-glr-gallery-event-handling-ginput-manual-clabel-sgskip-py)?

Reply:
Yes, on all of them. I have the feeling that we should first tell what the example is about and only after that discuss the technicalities in the note.

For the interactive functions, I'd place the note after the description sentence and right before the imports.

Reply:
Done.